server_admin/topics/users/ref-user-credentials.adoc

```diff
@@ -8,12 +8,18 @@
 You can manage credentials of a user in the *Credentials* tab. 
 
 .Credential management
-image:{project_images}/user-credentials.png[]
+image:{project_images}/user-credentials.png[user credentials]
```

Comment:
Can you replace the screenshot with one that has two or more credentials?

Reply:
But this is the credential screen when you just created a user. I think it doesn't make sense to have something different in this case, the original screenshot also didn't have any credentials